src/pages/Login.jsx

```diff
@@ -45,7 +45,7 @@ export default function Login() {
     setLoading(true);
     login({ email, password }).then((data) => {
       setLoading(false);
-      if (!data.success) {
+      if (!data.statusCode === 200) {
```

Comment:
(suggestion; non-blocking): We can use an enum to replace magic numbers. For example:

```
export const STATUS_CODE = {
  HTTP_OK: 200
};

if (!data.statusCode === STATUS_CODE.HTTP_OK) { ... }
```